install/0000_00_cluster-version-operator_00_namespace.yaml

```diff
@@ -10,3 +10,6 @@ metadata:
     name: openshift-cluster-version
     openshift.io/run-level: "1"
     openshift.io/cluster-monitoring: "true"
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/audit: privileged
+    pod-security.kubernetes.io/warn: privileged
```

Comment:
Reading [the KEP][1], it's not clear to me what the benefit of setting `warn` and `enforce` is.  Perhaps it's something that's useful in the short term while [the web-hook is in audit-only mode][2]?  But something we can drop once `enforce` grows some teeth?

[1]: https://github.com/kubernetes/enhancements/tree/3133297720aef516aa07a6553b314de2dd340d72/keps/sig-auth/2579-psp-replacement#api
[2]: https://github.com/openshift/enhancements/pull/899/files#diff-d6c490f9ea419e79a03c81cfe8ff642e110a202bdeeac5574d576dab666717ebR69